### ISSUE TITLE
[BUG] - PSelectWrapper SSR Accessibility Issue

### ISSUE THREAD

**Issue by anjuliehaaser** [porsche-design-system/porsche-design-system#2855]
### Contact Details

anjulie.haaser@porsche.digital

### JS Framework

React, NextJS App Router

### Porsche Design System version

3.7.0

### What browsers are you seeing the problem on?

Chrome

### Browser version

118.0.5993.96 

### Reproduction Example

https://ln9tgq.stackblitz.io

### Steps to reproduce the behavior

1. Open Dev Tools
2. Compare Code coming from server with rendered code
3. Dropdown Button coming from server is missing necessary aria-roles and an accessible name which is causing CLI Lighthouse ("@lhci/cli": 0.12.0) Accessibility Errors.

### Expected behavior

We would expect the dropdown button to have necessary aria-roles and accessible name when coming from server.

This seems to be a SSR issue since the button coming from the server looks like <button type="button" role="combobox" id="value"> and is missing aria-attributes.
The button that gets rendered has the aria-attributes.

### Relevant log output

```shell
When running the lighthouse tests in the CLI the following two lighthouse errors are thrown:

"ARIA Error: [role]s do not have all required [aria-*] attributes"
"NAMES AND LABELS Error: Buttons do not have an accessible name"

Failing elements (dropdown button):
button#value
<button type="button" role="combobox" id="value">
```

**Comment by o-hook** [comment]
Hi @anjuliehaaser, 
thanks for reporting. If I understood you correctly, you test the **SSR** variants of the components against accessibility issues **before** they were bootstrapped by the **client**? 
In this case, it is quite obvious that accessibility issues can occur if tested automatically with e.g. Lighthouse. But they will never have a negative effect to the user, because the user always interacts with the bootstrapped version which comes with all the functionalities and accessibility improvements. The SSR variant is just a visual presentation to get a faster visual rendering experience without any functionality or accessibility improvements.

So in your case, you should deactivate the accessibility test in Lighthouse completely for the SSR components, but for sure, you should test the components (and all other modules and pages) when they are bootstrapped. And then you will get HTML which is rendered correctly and should look something like this:
```
<button type="button" role="combobox" id="value" aria-labelledby="label" aria-haspopup="listbox" aria-expanded="false" aria-controls="list"></button>
<div class="sr-text" id="label">Option A, Some label</div>
```
